### PR TITLE
Refactor how we do search and navbar on the library

### PR DESCRIPTION
**Pull request by jacksonh**
- Remove text to speech feature flag, its enabled for everyone now
- Use a new modal for search

### CODE REVIEW

**Comment by jacksonh** [comment]
@satindar I cleaned this up and sort of got firstResponder working the way I wanted with introspection.

**Comment by jacksonh** [comment]
@satindar I think this one is ready for review now.

File: apple/OmnivoreKit/Sources/Models/LinkedItemFilter.swift [hunk 1]
```diff
@@ -6,6 +6,7 @@ public enum LinkedItemFilter: String, CaseIterable {
   case newsletters
   case all
   case archived
+  case hasHighlights
```
Comment (satindar) [inline apple/OmnivoreKit/Sources/Models/LinkedItemFilter.swift:9]: Nice. This will be nice for testing highlights! (and for users of course)

**Review by satindar — APPROVED** [review]

**Comment by satindar** [comment]
I'll do some more testing on this and merge it in unless something is broken. Then rebase the mac update branch over this.

**Pull request merged**